tooling/noir_js/test/node/e2e.test.ts

```diff
@@ -69,19 +69,7 @@ it('end-to-end proof creation and verification (inner)', async () => {
   expect(isValid).to.be.true;
 });
 
-// The "real" workflow will involve a prover and a verifier on different systems.
-//
-// We cannot do this in our tests because they will panic with:
-// `unreachable`
-//
-// This happens when we we create a proof with one barretenberg instance and
-// try to verify it with another.
-//
-// If this bug is fixed, we can remove this test and split barretenberg into
-// a prover and verifier class to more accurately reflect what happens in production.
```

Comment:
Can you add a TODO? Seems like we can tackle this now.

Reply:
I'll create an issue instead, TODO will be buried